EnglishFilter/sections/specific.txt

```diff
@@ -237,6 +237,9 @@ xxxparodyhd.net,movgotv.com,123kmovies.net,123movies.*,bluemovies.co,f5movies.to
 ||ytboob.com/long-limit-$script
 ||impactserving.com/preroll.engine$xmlhttprequest,redirect=nooptext
 ||twinrdsrv.com/preroll.engine$xmlhttprequest,redirect=nooptext
+3dmodelshare.org##.post > div.single-box
+3dmodelshare.org##.homeadv > span
```

Comment:
I don't see it

Reply:
![Screenshot 2020-09-01 at 11 32 05](https://user-images.githubusercontent.com/18536328/91826873-d2fec800-ec46-11ea-96f3-a1e73a078158.png)

Reply:
Ok